server/__init__.py

```diff
@@ -253,7 +253,7 @@ def create_app():
     register_routes_custom_dc(app)
 
   register_routes_base_dc(app)
-  if cfg.SHOW_DISASTER or os.environ.get('ENABLE_MODEL') == 'true':
+  if cfg.SHOW_DISASTER:
```

Comment:
could there be a case where SHOW_DISASTER is false even though ENABLE_MODEL is true? because we need disaster routes to be registered for nl interface

Reply:
I wonder which instance have that requirement? And for such cases, we should set SHOW_DISASTER=true explicitly?

Reply:
Revert this but we definitely need to separate the two configs

Reply:
actually moved the disaster api to base route now and remove the ENABLE_MODEL check